app/controllers/api/v1/profiles_controller.rb

```diff
@@ -6,4 +6,21 @@ def show
       format.yaml { render yaml: @user }
     end
   end
+
+  def me
+    authenticate_or_request_with_http_basic do |username, password|
+      if (user = AuthenticatedUser.authenticate(username.strip, password))
```

Comment:
:thinking: Are those brackets around assignment enforced by Rubocop?

Is `if user = AuthenticatedUser.authenticate(username.strip, password)` possible as well?

Reply:
It is indeed enforced by Rubocop. It wants the parentheses to make it clear that we didn't accidentally just use `=` instead of `==`. The parentheses signify that we meant to do assignment instead of checking the value in the if

Reply:
Makes sense. Speaking of rubocop :police_officer: , currently it is complaining in new User model payload.

```
Offenses:

app/models/authenticated_user.rb:6:17: C: [Correctable] Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://rubystyle.guide#spaces-braces)
    super.merge({"mfa" => mfa_level})
                ^
app/models/authenticated_user.rb:6:36: C: [Correctable] Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://rubystyle.guide#spaces-braces)
    super.merge({"mfa" => mfa_level})
```

Reply:
oh thanks good eye! I missed that one